src/app/campaign.service.ts

```diff
@@ -190,6 +191,10 @@ export class CampaignService {
     return of([
     ]);
   }
+
+  getCampaignImpactStats(): Observable<CampaignStats>{
```

Comment:
if you want to be able to show this in a template right now without waiting for the sf part to be done you could temporarily add a line like:
```ts
    return of(new CampaignStats(42, 65)); // 
```

then you just have to delete that line when we're ready to actually make the get request to SF.

Reply:
Yep – there are also examples in the repo which are closely analogous like `CampaignResolver`. (Although for a value used once in the app we would probably want to simplify by removing `TransferState` stuff, which I think is a per-session cache. https://levelup.gitconnected.com/a-beginners-guide-to-implementing-transferstate-in-angular-a313f3244f03)